.github/scripts/run_tests.sh

```diff
@@ -0,0 +1,108 @@
+set -ex
+
+function run_torch_xla_python_tests() {
+  PYTORCH_DIR=$1
+  XLA_DIR=$2
+  USE_COVERAGE="${3:-0}"
+
+  pushd $XLA_DIR
+    echo "Running Python Tests"
+    if [ "$USE_COVERAGE" != "0" ]; then
+      pip install coverage==6.5.0 --upgrade
+      pip install coverage-lcov
+      pip install toml
+      ./test/run_tests.sh
+      coverage combine
+      mkdir lcov && cp .coverage lcov/
+      coverage-lcov --data_file_path lcov/.coverage
+      coverage html
+      cp lcov.info htmlcov/
+      mv htmlcov ~/
+      chmod -R 755 ~/htmlcov
+    else
+      ./test/run_tests.sh
+    fi
+  popd
+}
+
+function run_torch_xla_cpp_tests() {
+  PYTORCH_DIR=$1
+  XLA_DIR=$2
+  USE_COVERAGE="${3:-0}"
+
+  TORCH_DIR=$(python -c "import pkgutil; import os; print(os.path.dirname(pkgutil.get_loader('torch').get_filename()))")
+  export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${TORCH_DIR}/lib
+  if [ -x "$(command -v nvidia-smi)" ]; then
+    CUDA_PLUGIN_DIR=$(python -c "import pkgutil; import os; print(os.path.dirname(pkgutil.get_loader('torch_xla_cuda_plugin').get_filename()))")
+    export PJRT_LIBRARY_PATH=$CUDA_PLUGIN_DIR/lib/pjrt_c_api_gpu_plugin.so
+    export PJRT_DEVICE=LIBRARY
+    export PJRT_DYNAMIC_PLUGINS=1
+  else
+    export PJRT_DEVICE=CPU
+  fi
+  export XLA_EXPERIMENTAL="nonzero:masked_select"
```

Comment:
can you also add `nms`, example in https://github.com/pytorch/xla/blob/023e2c83dcf20b973ad4ec60b27469fcada02af5/test/tpu/run_tests.sh#L14-L15

Reply:
Done